### PR TITLE
Add User Feedback to Blazor Admin System Analytics for AB#13682 and AB#13683

### PR DESCRIPTION
**Pull request by BrianMaki**
# Implements [AB#13683](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13683).

## Description

Also implemented backend code for [AB#13682](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13682).

1. Add 'User Feedback' download button Blazor Admin System Analytics page
2. Updated System Analytics client razor code
3. Updated System Analytics store code to download 'User Feedback'
4. Updated Admin Server backend code to call CSV Helper to create csv file stream

Deployed to mock.

https://mock-admin.healthgateway.gov.bc.ca/analytics

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

Yes

Screen Shot 2022-08-16 at 9 36 34 AM

Screen Shot 2022-08-16 at 10 11 17 AM

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/Admin/Client/Services/IAnalyticsApi.cs [hunk 1]
```diff
@@ -69,4 +69,11 @@ public interface IAnalyticsApi
     /// <returns>HttpResponseMessage.</returns>
     [Get("/GetInactiveUsers?inactiveDays={inactiveDays}&timeOffset={timeOffset}")]
     Task<HttpResponseMessage> GetInactiveUsers(int inactiveDays, int timeOffset);
+
+    /// <summary>
+    /// Retrieves a list of User Feedback.
+    /// </summary>
+    /// <returns>HttpResponseMessage.</returns>
+    [Get("/GetUserfeedback")]
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Services/IAnalyticsApi.cs:77]: Nit: casing doesn't match controller definition `"/GetUserFeedback"`.

**Review by MikeLyttle — APPROVED** [review]

**Review by rayking29 — DISMISSED** [review]

**Comment by MikeLyttle** [comment]
@BrianMaki How does the library handle user comments that may include commas as well as quotation marks?

**Comment by BrianMaki** [comment]
> @BrianMaki How does the library handle user comments that may include commas as well as quotation marks?

Confirmed commas are handled.

Screen Shot 2022-08-16 at 11 27 13 AM

File: Apps/Admin/Client/Services/IAnalyticsApi.cs (hunk 1 above)
Comment (BrianMaki) [reply Apps/Admin/Client/Services/IAnalyticsApi.cs]: Fixed.

**Review by MikeLyttle — APPROVED** [review]

**Review by rayking29 — APPROVED** [review]

**Pull request merged**